hail/python/hail/backend/service_backend.py

```diff
@@ -135,17 +135,6 @@ def __repr__(self):
         return f'GitRevision({self.revision})'
 
 
-def _get_jar_specification(jar_url: Optional[str]) -> JarSpec:
-    user_config = get_user_config()
-
-    jar_url = jar_url or os.environ.get('HAIL_JAR_URL')
```

Comment:
I think this is the only env var which doesn't conform with the `HAIL_{SECTION}_{OPTION}` and should be `HAIL_QUERY_JAR_URL`. Seems a fine breaking change to make but I think you need to update it in Makefiles and the like.